contrib/format-iceberg/src/main/java/org/apache/drill/exec/store/iceberg/plan/IcebergPluginImplementor.java

```diff
@@ -99,6 +99,9 @@ public boolean canImplement(Filter filter) {
       try {
         TableScan tableScan = DrillRelOptUtil.findScan(filter.accept(SubsetRemover.INSTANCE));
         DrillTable drillTable = DrillRelOptUtil.getDrillTable(tableScan);
```

Comment:
Please also add a null check for `tableScan` to fix https://issues.apache.org/jira/browse/DRILL-8058.

Reply:
Good point. I'll give it a try.